### ISSUE TITLE
how are normalized parameters calculated?

### ISSUE THREAD

**Issue by rongliangzi** [leeyeehoo/CSRNet-pytorch#33]
Thanks for the released code!
we can see there are normalization in both train and test as follows:
```
transforms.Compose([
                       transforms.ToTensor(),transforms.Normalize(mean=[0.485, 0.456, 0.406],
                                     std=[0.229, 0.224, 0.225]),
                   ])
```
so why do we need normalization? how are the parameters 'mean' & 'std' calculated?

**Comment by rongliangzi** [comment]
I get it. It's illustrated in pytorch tutorial as [here](https://pytorch.org/docs/master/torchvision/models.html)
